user-registration-app/src/app/views/live-view/live-view.component.html

```diff
@@ -305,279 +303,427 @@
   </div>
   <div id="workshop-container" class="container-fluid nopadding">
     <div class="row no padding">
-      <div class="col-md-6 col-lg-3 nopadding">
+      <div class="col-md-6 col-lg-4 nopadding">
         <span class="workshop-track">Beginner</span>
         <div class="container-fluid py-4">
           <div class="col-12 p-md-0 px-4">
             <div class="card beginner">
-              <div class="card-image waves-effect waves-block waves-light">
+              <div class="card-image">
+                <img class="activator" src="../../../assets/images/Accuweather.jpeg">
+                <a class="waves-effect waves-light activator red btn">click to learn more</a>
+              </div>
+              <div class="card-content pt-5">
+                <span class="card-title activator" style="font-weight:bold;">AccuWeather: API Workshop</span>
+                <div style="font-weight:bold;">AccuWeather Team</div>
+                <div style="font-weight:bold;">Location: 125 Business Building (Workshop Room 2)</div>
+                <div style="font-weight:bold;">Time: 2:00 PM - 2:45 PM Saturday</div>
+              </div>
+              <div class="card-reveal">
+                <span class="card-title">AccuWeather: API Workshop<i
+                  class="material-icons right">close</i></span>
+                <p>General API Overview & AccuWeather Technologies</p>
+              </div>
+            </div>
+            <div class="card beginner">
+              <div class="card-image">
                 <img class="activator" src="../../../assets/images/workshop_leaders/Nathaniel_github.png">
-                <a class="btn halfway-fab waves-effect waves-light red activator">click to learn more</a>
+                <a class="waves-effect waves-light activator red btn">click to learn more</a>
               </div>
               <div class="card-content pt-5">
-                <span class="card-title activator" style="font-weight:bold;">Introduction to GitHub</span>
-                <div style="font-weight:bold;">Nathaniel Johnson, ACM</div>
-                <div style="font-weight:bold;">Location: 124 Business Building (Workshop Room 1)</div>
+                <span class="card-title activator" style="font-weight:bold;">Making Your Impact: The Nittany Ai Challenge</span>
+                <div style="font-weight:bold;">Brad Zdenek, NittanyAI</div>
```

Comment:
Does Nittany AI need a space between the words?

Reply:
and is the "i" in Ai supposed to be capitalized?

Reply:
Confirmed Nittany AI and Fixed

Reply:
does the i need to be capitalized? I guess?

Reply:
Fixed